From: Francesco Paolo Lovergine <frankie@debian.org>
Subject: add 'foreign' to AUTOMAKE_OPTIONS
Origin: vendor
Forwarded: no

---
 Makefile.am |    2 +-
 Makefile.in |    2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

--- a/Makefile.am
+++ b/Makefile.am
@@ -1,6 +1,6 @@
 ACLOCAL_AMFLAGS = -I m4
 
-AUTOMAKE_OPTIONS = dist-zip
+AUTOMAKE_OPTIONS = dist-zip foreign
 
 EXTRA_DIST = makefile.vc nmake.opt Android.mk mainpage.doxy README.coverage \
 	spatialite-sql-latest.html config-msvc.h
--- a/Makefile.in
+++ b/Makefile.in
@@ -355,7 +355,7 @@ top_build_prefix = @top_build_prefix@
 top_builddir = @top_builddir@
 top_srcdir = @top_srcdir@
 ACLOCAL_AMFLAGS = -I m4
-AUTOMAKE_OPTIONS = dist-zip
+AUTOMAKE_OPTIONS = dist-zip foreign
 EXTRA_DIST = makefile.vc nmake.opt Android.mk mainpage.doxy README.coverage \
 	spatialite-sql-latest.html config-msvc.h
 
