Description: Link libm for math symbols.
Author: Bas Couwenberg <sebastic@xs4all.nl>
--- a/src/Makefile.am
+++ b/src/Makefile.am
@@ -42,7 +42,7 @@ libspatialite_la_LDFLAGS = -version-info
 libspatialite_la_LIBADD += -ldl
 else
 libspatialite_la_LDFLAGS = -version-info 7:0:0
-libspatialite_la_LIBADD += -lpthread -ldl
+libspatialite_la_LIBADD += -lpthread -ldl -lm
 endif
 endif
 
--- a/src/Makefile.in
+++ b/src/Makefile.in
@@ -79,7 +79,7 @@ POST_UNINSTALL = :
 build_triplet = @build@
 host_triplet = @host@
 @ANDROID_TRUE@@MINGW_FALSE@am__append_1 = -ldl
-@ANDROID_FALSE@@MINGW_FALSE@am__append_2 = -lpthread -ldl
+@ANDROID_FALSE@@MINGW_FALSE@am__append_2 = -lpthread -ldl -lm
 @ANDROID_TRUE@@MINGW_FALSE@am__append_3 = -ldl
 @ANDROID_FALSE@@MINGW_FALSE@am__append_4 = -lpthread -ldl
 subdir = src
