From: David Paleino <dapal@debian.org>
Subject: fix missing #define in header
Origin: vendor
Forwarded: no

---
 src/headers/spatialite/gaiaexif.h |    1 +
 1 file changed, 1 insertion(+)

--- spatialite.orig/src/headers/spatialite/gaiaexif.h
+++ spatialite/src/headers/spatialite/gaiaexif.h
@@ -87,6 +87,7 @@ extern "C"
 #define GAIA_GEOMETRY_BLOB	8
 /** this BLOB does actually contain a TIFF image */
 #define GAIA_TIFF_BLOB		9
+#define GAIA_WAVELET_BLOB	10
 
 /* constants used for EXIF value types */
 /** unrecognized EXIF value */
