Index: spatialite/configure
===================================================================
--- spatialite.orig/configure	2012-02-23 13:42:29.000000000 +0100
+++ spatialite/configure	2012-02-27 12:25:37.000000000 +0100
@@ -17201,7 +17201,7 @@
     ac_res="none required"
   else
     ac_res=-l$ac_lib
-    LIBS="-l$ac_lib -lm -lgeos $ac_func_search_save_LIBS"
+    LIBS="-l$ac_lib $ac_func_search_save_LIBS"
   fi
   if ac_fn_c_try_link "$LINENO"; then :
   ac_cv_search_GEOSTopologyPreserveSimplify=$ac_res
@@ -17271,7 +17271,7 @@
     ac_res="none required"
   else
     ac_res=-l$ac_lib
-    LIBS="-l$ac_lib -lm -lgeos $ac_func_search_save_LIBS"
+    LIBS="-l$ac_lib $ac_func_search_save_LIBS"
   fi
   if ac_fn_c_try_link "$LINENO"; then :
   ac_cv_search_GEOSCoveredBy=$ac_res
Index: spatialite/configure.ac
===================================================================
--- spatialite.orig/configure.ac	2012-02-23 13:42:29.000000000 +0100
+++ spatialite/configure.ac	2012-02-27 12:25:14.000000000 +0100
@@ -124,7 +124,7 @@
 if test x"$enable_geos" != "xno"; then
   OMIT_GEOS_FLAGS=
   AC_CHECK_HEADERS(geos_c.h,, [AC_MSG_ERROR([cannot find geos_c.h, bailing out])])
-  AC_SEARCH_LIBS(GEOSTopologyPreserveSimplify,geos_c,,AC_MSG_ERROR(['libgeos_c' is required but it doesn't seem to be installed on this system.]),-lm -lgeos)
+  AC_SEARCH_LIBS(GEOSTopologyPreserveSimplify,geos_c,,AC_MSG_ERROR(['libgeos_c' is required but it doesn't seem to be installed on this system.]),)
   #-----------------------------------------------------------------------
   #   --enable-geosadvanced
   #
@@ -133,7 +133,7 @@
 	  [], [geosadvanced=yes])
   if test x"$enable_geosadvanced" != "xno"; then
 	  GEOSADVANCED_FLAGS=-DGEOS_ADVANCED
-	  AC_SEARCH_LIBS(GEOSCoveredBy,geos_c,,AC_MSG_ERROR([obsolete 'libgeos_c' (< v.3.3.0). please retry specifying: --disable-geosadvanced.]),-lm -lgeos)
+	  AC_SEARCH_LIBS(GEOSCoveredBy,geos_c,,AC_MSG_ERROR([obsolete 'libgeos_c' (< v.3.3.0). please retry specifying: --disable-geosadvanced.]),)
   else
 	  GEOSADVANCED_FLAGS=
   fi
