Index: spatialite/src/Makefile.am
===================================================================
--- spatialite.orig/src/Makefile.am	2012-03-02 11:53:36.000000000 +0100
+++ spatialite/src/Makefile.am	2012-03-02 11:59:04.000000000 +0100
@@ -27,7 +27,7 @@
 if MINGW
 libspatialite_la_LDFLAGS = -version-info 3:0:0 -no-undefined
 else
-libspatialite_la_LDFLAGS = -version-info 4:3:0
+libspatialite_la_LDFLAGS = -version-info 5:0:2
 libspatialite_la_LIBADD += -lpthread -ldl
 endif
 
Index: spatialite/src/Makefile.in
===================================================================
--- spatialite.orig/src/Makefile.in	2012-03-02 11:53:36.000000000 +0100
+++ spatialite/src/Makefile.in	2012-03-02 12:00:00.000000000 +0100
@@ -282,7 +282,7 @@
 	./spatialite/libsplite.la ./shapefiles/libshapefiles.la \
 	./srsinit/libsrsinit.la ./virtualtext/libvirtualtext.la \
 	$(am__append_1)
-@MINGW_FALSE@libspatialite_la_LDFLAGS = -version-info 4:3:0
+@MINGW_FALSE@libspatialite_la_LDFLAGS = -version-info 5:0:2
 @MINGW_TRUE@libspatialite_la_LDFLAGS = -version-info 3:0:0 -no-undefined
 MOSTLYCLEANFILES = *.gcna *.gcno *.gcda
 all: all-recursive
