Description: Link libxml2 for successfull test case build.
Author: Bas Couwenberg <sebastic@debian.org>
Last-Update: 2013-12-14
Forwarded: https://groups.google.com/d/msg/spatialite-users/kIZA-n4f9kk/IoaxSGjUWZEJ

--- a/test/Makefile.am
+++ b/test/Makefile.am
@@ -89,7 +89,7 @@ check_PROGRAMS += \
 endif
 
 AM_CFLAGS = -I@srcdir@/../src/headers -I@srcdir@
-AM_LDFLAGS = -L../src -lspatialite -lm $(GCOV_FLAGS)
+AM_LDFLAGS = -L../src -lspatialite -lm -lxml2 $(GCOV_FLAGS)
 
 TESTS = $(check_PROGRAMS)
 
