Description: Link libpthread to fix check_multithread build failure 
Author: Bas Couwenberg <sebastic@debian.org>
--- a/test/Makefile.am
+++ b/test/Makefile.am
@@ -87,7 +87,7 @@ check_PROGRAMS += \
 endif
 
 AM_CFLAGS = -I@srcdir@/../src/headers -I@srcdir@
-AM_LDFLAGS = -L../src -lspatialite -lm -lxml2 $(GCOV_FLAGS)
+AM_LDFLAGS = -L../src -lpthread -lspatialite -lm -lxml2 $(GCOV_FLAGS)
 
 TESTS = $(check_PROGRAMS)
 
--- a/test/Makefile.in
+++ b/test/Makefile.in
@@ -972,7 +972,7 @@ top_builddir = @top_builddir@
 top_srcdir = @top_srcdir@
 AM_CPPFLAGS = @CFLAGS@ @GEOS_CFLAGS@ @LIBXML2_CFLAGS@
 AM_CFLAGS = -I@srcdir@/../src/headers -I@srcdir@
-AM_LDFLAGS = -L../src -lspatialite -lm $(GCOV_FLAGS)
+AM_LDFLAGS = -L../src -lpthread -lspatialite -lm $(GCOV_FLAGS)
 TESTS = $(check_PROGRAMS)
 MOSTLYCLEANFILES = *.gcna *.gcno *.gcda
 EXTRA_DIST = asprintf4win.h \
