Description: Fix spelling errors.
 * updgrade -> upgrade
Author: Bas Couwenberg <sebastic@debian.org>

--- a/src/spatialite/table_cloner.c
+++ b/src/spatialite/table_cloner.c
@@ -2305,7 +2305,7 @@ gaiaAuxClonerExecute (const void *handle
 	  if (!upgrade_output_table (cloner))
 	    {
 		spatialite_e
-		    ("CloneTable: unable to updgrade the output table \"%s\"\n",
+		    ("CloneTable: unable to upgrade the output table \"%s\"\n",
 		     cloner->out_table);
 		return 0;
 	    }
--- a/src/shapefiles/validator.c
+++ b/src/shapefiles/validator.c
@@ -2288,7 +2288,7 @@ sanitize_geometry_column_common (const v
     if (report->n_not_repaired > 0)
       {
 	  fprintf (out,
-		   "\t\t\t<tr><td colspan=\"2\" class=\"err\">This layer contains severly damaged Geometries (beyond any possible repair);<br>");
+		   "\t\t\t<tr><td colspan=\"2\" class=\"err\">This layer contains severely damaged Geometries (beyond any possible repair);<br>");
 	  fprintf (out,
 		   "please manually check the <b>%s</b> table.</td></tr>\n",
 		   tmp_table);
